@here/harp-webtile-datasource/lib/WebTileDataSource.ts

```diff
@@ -121,23 +127,28 @@ export class WebTileDataSource extends DataSource {
                 texture.magFilter = THREE.LinearFilter;
                 texture.generateMipmaps = false;
                 tile.addOwnedTexture(texture);
-                const transparent =
-                    this.m_options.renderingOptions !== undefined &&
-                    this.m_options.renderingOptions.transparent === true;
-                const opacity =
-                    this.m_options.renderingOptions !== undefined &&
-                    this.m_options.renderingOptions.opacity !== undefined
-                        ? this.m_options.renderingOptions.opacity
-                        : 1;
+
+                let transparent = false;
+                let opacity = 1;
+                let renderOrder = 0;
+                if (this.m_options.renderingOptions !== undefined) {
+                    opacity = this.m_options.renderingOptions.opacity ?? 1;
+                    transparent =
+                        this.m_options.renderingOptions.transparent === true ||
+                        (opacity !== undefined && opacity < 1);
+                    renderOrder = this.m_options.renderingOptions.renderOrder ?? 0;
+                }
                 const material = new THREE.MeshBasicMaterial({
                     map: texture,
-                    depthTest: false,
-                    depthWrite: false,
                     opacity,
-                    transparent: transparent || (opacity !== undefined && opacity < 1.0)
+                    transparent,
+                    polygonOffset: renderOrder > 0 ? true : false,
+                    polygonOffsetFactor: -1,
```

Comment:
Why not keeping the `depthTest: false`. Especially when combining satellite + omv this could cause issues.

Reply:
ok, done

Reply:
I actually meant to use `depthTest: false` instead of `polgonOffset` to not pollute the depth buffer with "wrong" values

Reply:
Yes, agreed, but the thing is, that with pure renderOrder we cannot interleave correctly transparent and non-transparent layers, as they are in different render groups. It´s probably a not so common case to put a opaque layer on top of a transparent one, but still it would not handle it as expected. 
I could somehow make it optional to use either or, but this would again blow up the API..

Reply:
We should keep all objects in the opaque queue as we also do for the OMV layers.
We have a utility method to enable blending even if `transparent=false`:
https://github.com/heremaps/harp.gl/blob/master/@here/harp-materials/lib/Utils.ts#L81

Reply:
ok, nice. I ll adjust to that then